osfoffline/gui/qt/preferences.py

```diff
@@ -148,13 +149,14 @@ def selector(self, selected_index):
 
             self._executor = QtCore.QThread()
             self.node_fetcher = NodeFetcher()
-            self.treeWidget.setCursor(QtCore.Qt.BusyCursor)
+            QApplication.setOverrideCursor(QtCore.Qt.WaitCursor)
```

Comment:
Out of curiosity, will the application cursor get "stuck" in a waiting mode if the node-fetcher operation fails? (eg L207)